### DISCUSSION TITLE
Feature: emoji flag icons for languages

### DISCUSSION THREAD

**Discussion by Linguisto** [directus/directus#2691] [Feature Requests]
It'd be very great to add language emojis with flags on all Language interface inputs :flags:

**Comment by benhaynes** [comment]
I agree! Our new v9 Translations interface will allow you to add your own locale keys/options... so you can paste those flags right in!

**Reply by benhaynes** [reply]
In fact, you can already do this in v8 by pasting emojis into the Translation Options! 😄 

![Screen Shot 2020-04-30 at 9 35 26 AM](https://user-images.githubusercontent.com/522079/80716711-fc0b2280-8ac5-11ea-821a-68f7788e16df.png)

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
